packages/auth-construct/src/construct.test.ts

```diff
@@ -545,6 +545,34 @@ void describe('Auth construct', () => {
     );
   });
 
+  void it('configures Cognito to send emails with SES when senders field is populated', () => {
+    const app = new App();
+    const stack = new Stack(app);
+    const expectedEmail = 'Example.com <noreply@example.com>';
+    const expectedReply = 'support@example.com';
+    const sesEmailSettings = {
+      fromEmail: 'noreply@example.com',
+      fromName: 'Example.com',
+      replyTo: 'support@example.com',
+    };
+    new AmplifyAuth(stack, 'test', {
+      loginWith: {
+        email: true,
+      },
+      senders: {
+        email: sesEmailSettings,
+      },
+    });
+
+    const template = Template.fromStack(stack);
+    template.allResourcesProperties('AWS::Cognito::UserPool', {
+      EmailConfiguration: {
+        From: expectedEmail,
+        ReplyToEmailAddress: expectedReply,
```

Comment:
can you validate the fromName value as well?

Reply:
yep!

Reply:
The From field of EmailConfiguration validates both fromName and fromEmail, the variable expectedEmail is structured "fromName <fromEmail>". In EmailConfiguration, there is not a separate field that is only for fromName or a separate field for fromEmail, attached is the properties related to EmailConfiguration highlighted in red.
<img width="572" alt="email-template" src="https://github.com/user-attachments/assets/5d04ad66-99fc-4f65-8205-79cd0ccb2102">
 I can update the variable name to make this more clear.

Reply:
oh i see